lib/RestClient.ts

```diff
@@ -9,7 +9,8 @@ import util = require("./Util");
 
 export interface IRestResponse<T> {
     statusCode: number,
-    result: T | null
+    result: T | null,
+    headers: Object
```

Comment:
I wonder if we should make this of type [IHeaders](https://github.com/Microsoft/typed-rest-client/blob/master/lib/Interfaces.ts#L7)?

Reply:
Genuine question... I am not sure :)

Reply:
I originally went with object because that's what it is [here](https://nodejs.org/api/http.html#http_message_headers). Additionally, I don't really see a difference between using IHeaders and Object - isn't a JavaScript object just a collection of string keys mapping to values (aka an IHeader)? I don't think any additional info is conveyed there, all we're really doing by calling it a IHeader is making the user look one more place for a type definition. If anything I'd be more partial to just doing away with the IHeaders type (though we probably shouldn't actually do that because of back-compat).